velox/functions/prestosql/types/fuzzer_utils/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+velox_add_library(velox_presto_types_fuzzer_utils
+                  TimestampWithTimeZoneInputGenerator.cpp)
+
+velox_link_libraries(
```

```suggestion
add_library(velox_presto_types_fuzzer_utils
                  TimestampWithTimeZoneInputGenerator.cpp)

target_link_libraries(
```

A fuzzer util library doesn't need to be part of the main libvelox.

---

I tried to, but (correct me if I'm wrong) since the AbstractInputGenerator is part of the CustomTypeFactories, TimestampWithTimeZoneRegistration.cpp needs to link against TimestampWithTimeZoneInputGenerator.h in order to construct the TimestampWithTimeZoneInputGenerator, so if it's not included in the main libvelox we run into linker errors.  Similarly velox/common/fuzzer is included in the main libvelox and it contains ConstrainedGenerators.h which is just a bunch of fuzzers (AbstractInputGenerators).

---

I thought we moved `velox/common/fuzzer` out recently but I'm probably mixing something up. Hm, i'll have a look.